examples/l3gd20/main.c

```diff
@@ -0,0 +1,36 @@
+/* vim: set sw=2 expandtab tw=80: */
+
+#include <l3gd20.h>
+#include <math.h>
+#include <stdio.h>
+#include <timer.h>
+
+#define ABS(N) ((N < 0) ? (-N) : (N))
```

Comment:
This already pulls in math.h, why not just use `fabs`?

Reply:
Done